### PR TITLE
CLOUDP-126263: Create command “atlas backups exports jobs describe”

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes

Create command “atlas backups exports jobs describe”

_Jira ticket:_ CLOUDP-126263

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: docs/atlascli/command/atlas-backups-exports-jobs-describe.txt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --bucketId
+     - string
+     - true
+     - Unique identifier of the role that Atlas can use to access the bucket.
```
Comment (sarahsimpers) [inline docs/atlascli/command/atlas-backups-exports-jobs-describe.txt:38]: This seems like the wrong description here (or it's the wrong flag on line 35)

**Review by sarahsimpers — CHANGES_REQUESTED** [review]
Just one copy change and a place where it appears to be the wrong description

File: docs/atlascli/command/atlas-backups-exports-jobs-describe.txt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Describe cloud backup export job for your project, cluster and bucket.
```
Comment (sarahsimpers) [inline docs/atlascli/command/atlas-backups-exports-jobs-describe.txt:15]: Oops not sure why this didn't come through with the last review
```suggestion
Return all cloud backup export jobs for your project, cluster, and bucket.
```

**Review by drinkbird — DISMISSED** [review]

**Review by sarahsimpers — DISMISSED** [review]

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**